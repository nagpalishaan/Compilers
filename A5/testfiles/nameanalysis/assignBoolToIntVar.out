Error at line 3: expected type does not match. Type: lang.ast.BoolType@c86b9e3 != Expected type: lang.ast.IntType@687e99d8
Error at line 3: incorrect types for AssignVar
