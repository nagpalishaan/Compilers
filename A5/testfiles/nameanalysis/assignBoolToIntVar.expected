Error at line 3: expected type does not match. Type: lang.ast.BoolType@1dac5ef != Expected type: lang.ast.IntType@6ab778a
Error at line 3: incorrect types for AssignVar
